Source/santactl/Commands/SNTCommandStatus.m

```diff
@@ -229,6 +233,14 @@ - (void)runWithArguments:(NSArray *)arguments {
       printf("  %-25s | %lld\n", "Root cache count", rootCacheCount);
       printf("  %-25s | %lld\n", "Non-root cache count", nonRootCacheCount);
     }
+
+    printf("  %-25s | %d\n", "USB Blocking", configurator.blockUSBMount);
+
+    if (configurator.blockUSBMount && [configurator.remountUSBMode count] > 0) {
```

Comment:
Prefer using the property version of count: `configurator.remountUSBMode.count`

Reply:
Done. I've also changed it to print "Yes" or "No" instead.